lib/root_controller.js

```diff
@@ -0,0 +1,174 @@
+var Tessel = require('./tessel/tessel');
+var logs = require('./logs');
+var debug = require('debug')('controller');
+
+// create an object for t2 root steps for beeing more handy about unit testing of the T2 root command itself.
+var controller = {};
+controller.ssh = {
+  makeMenu: function(tessel, index, rtm) {
+    if (tessel.connection.connectionType === 'LAN') {
+      if (!tessel.connection.authorized) {
+        rtm.add(index + ') ' + tessel.name + ': ' + tessel.connection.ip + ' (not authorized) \n');
+      } else {
+        rtm.add(index + ') ' + tessel.name + ': ' + tessel.connection.ip + ' (authorized) \n');
+      }
+
+    } else if (tessel.connection.connectionType === 'USB') {
+      if (!tessel.connection.authorized) {
+        rtm.add(index + ') ' + tessel.name + ': [USB] (not authorized) \n');
```

Comment:
:bulb: ... ? If I steal the Professors Tessel 2 for installing a root kit .... I simple need to connect it to USB ? COOL :smiling_imp: